### ISSUE TITLE
Need to add able add nodejs pools

### ISSUE THREAD

**Issue by 1rV1N-git** [turtlecoin/turtlecoin-pools-json#11]
Like https://trtl.cryptopool.space

**Comment by derp-derp-derp** [comment]
We're working on versioning the list to a new version that contains forknote and others.

Do the Node.js pools have an API like forknote?

**Comment by derp-derp-derp** [comment]
Suggestions on the best way to organize the JSON object(s) in this new file are welcome as well.

**Comment by derp-derp-derp** [comment]
Please see here and comment accordingly: [](https://github.com/turtlecoin/turtlecoin-pools-json/pull/12)

**Comment by 1rV1N-git** [comment]
https://cryptonote-pool-api.restlet.io/

**Comment by derp-derp-derp** [comment]
That's a link to the API docs. I think we're more interested in the endpoint URL in this case, which is https://api.xmrpool.net - although I don't see how either of those tie back to the TRTL pool specifically.

**Comment by derp-derp-derp** [comment]
Nice.

**Comment by yamiM0NSTER** [comment]
V2 could use something like that:
```
"atpool.party" : {
            "url" : "https://turtle.atpool.party/",
            "api" : "http://turtle-eu.atpool.party:8117/"
},
```
And the split addition looks good on top of that.

**Comment by derp-derp-derp** [comment]
How does this look? [/turtlecoin-pools-json/pull/12/files#diff-1e9f9ce0dc6bcf33b42c76b073282a64](https://github.com/turtlecoin/turtlecoin-pools-json/pull/12/files#diff-1e9f9ce0dc6bcf33b42c76b073282a64)

**Comment by derp-derp-derp** [comment]
fixed in #12
